Automattic/vip-cli
# Using new VIP Docker images for dev-env

## Description

This PR changes the Docker images that are used in the developer environment to the ones in https://github.com/Automattic/vip-container-images. 

Some other relevant changes:

- Removed the option to fetch Jetpack from a remote image.
- Removed the option to run different PHP Versions (fixed to 7.4 for now).
- Hardcoded the supported WordPress versions.

## Steps to Test

1. Check out PR and install `vip` from it.
2. Developer Environment should use the new images and work normally.
